nightfall-deployer/contracts/ChallengesUtil.sol

```diff
@@ -76,9 +76,6 @@ library ChallengesUtil {
                 transaction.recipientAddress != ZERO ||
                 transaction.commitments[0] == ZERO ||
                 transaction.commitments[1] != ZERO ||
-                transaction.commitments.length != 1 ||
-                transaction.nullifiers.length != 0 ||
-                transaction.compressedSecrets.length != 0 ||
```

Comment:
Do we now need to check that the `compressedSecrets` array is full of `0x00..00`s

Reply:
Added all zero compressedSecrets check